x/denommetadata/proposal_handler.go

```diff
@@ -21,29 +22,37 @@ func NewDenomMetadataProposalHandler(k types.BankKeeper) govtypes.Handler {
 }
 
 // HandleCreateDenomMetadataProposal is a handler for executing a passed denom metadata creation proposal
-func HandleCreateDenomMetadataProposal(ctx sdk.Context, k types.BankKeeper, p *types.CreateDenomMetadataProposal) error {
+func HandleCreateDenomMetadataProposal(ctx sdk.Context, k *keeper.Keeper, p *types.CreateDenomMetadataProposal) error {
 
 	for _, metadata := range p.TokenMetadata {
-		found := k.HasDenomMetaData(ctx, metadata.Base)
+		found := k.GetBankKeeper().HasDenomMetaData(ctx, metadata.Base)
```

Comment:
As you already added a keeper I would simply move this logic there vs calling GetBankKeeper()

Reply:
moved